### ISSUE TITLE
(for Thunkable X) Calling functions through a variable

### ISSUE THREAD

**Issue by AlexITMan** [thunkable/thunkable-issues#190]
Now to call a function, you must use the block of this function with its name. It would be nice to call the desired function, passing the name to the call block.

![изображение](https://user-images.githubusercontent.com/20015046/61297036-0d5b9a80-a7e4-11e9-80ba-3ee773238f36.png)

**Comment by mark-friedman** [comment]
I agree that would be useful.  Note that I am assuming that what you want is to be able to pass in a block representing a method of a component or a variable, parameter, or other block whose value is a method.

I've also changed the title of this issue to reflect the fact that you want to be able to call component methods via a variable.  If you want to also call functions via a variable you'll need to create another feature request ;-)

**Comment by AlexITMan** [comment]
Thank you, Mark, you understood perfectly.
